### PR TITLE
Further website updates

### PR DESCRIPTION
**Pull request by jlstevens**
This PR is currently WIP and aims to greatly improve the content of the EarthML website as well as the presentation of the material (the new topic notebooks and the user guides). Here is a current screenshot of the updated homepage:

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,93 @@
...
+>     performance at scale.
+
+A major goal of this website is to demonstrate examples of how Python
+tools may be used to perform machine learning tasks in the earth
+sciences. A number of such examples are available in the
+[Topics](topics) section:
+
+<div>
+<div >
+  <a href="http://earthml.pyviz.org/topics/trees.html">
```
Comment (jsignell) [inline README.md]: This link is not quite right.

Comment (jlstevens) [reply README.md]: Thanks. I think this is part of how `nbsite` is being used.

File: doc/index.rst [hunk 2]
```diff
@@ -49,3 +124,54 @@ To be completed.
...
+   ********
+
+   **Lightweight Python tools for environmental simulation**
+
+   EarthSim is a project for developing Python-based workflows for
+   specifying, launching, visualizing, and analyzing environmental
+   simulations, such as those for hydrology modeling. EarthSim is
+   designed to be a lightweight project internally, relying on code and
+   documentation primarily maintained in other, freely available,
+   general-purpose SciPy and `PyViz <http://pyviz.org>`_ projects:
```
Comment (jbednar) [inline doc/index.rst]: I assume this is meant for me to update later?  Seems like you could at least replace EarthSim with EarthML and do some minimal updates...

File: doc/index.rst [hunk 3]
```diff
@@ -49,3 +124,54 @@ To be completed.
...
+      How to install the code, notebooks, and data files needed for the examples.
+   - `User Guide <user_guide>`_:
+      Usage information for the ``earthsim`` Python package, which
+      contains a small amount of code specific to environmental
+      simulation such as helper functions for annotating plots and
+      specifying irregular meshes.
+   - `Topics <topics>`_:
+      A set of Jupyter notebooks that show how to use the various PyViz
+      tools together to solve earth-science problems, such as how to run
+      various simulators.
```
Comment (jbednar) [inline doc/index.rst]: Do we have those three sections?

File: doc/index.rst [hunk 4]
```diff
@@ -49,3 +124,54 @@ To be completed.
...
+
+   If you have any notebooks showing how to use these tools to do useful
+   work in environmental simulation, we'd love to add them to the Topics!
+
+   Please feel free to report `issues
+   <https://github.com/pyviz/earthsim/issues>`_ or `contribute code.
+   <https://help.github.com/articles/about-pull-requests>`_
```
Comment (jbednar) [inline doc/index.rst]: earthsim->earthml

File: environment.yml [hunk 5]
```diff
@@ -0,0 +1,245 @@
...
+  - pyviz=0.9.13a0=py_0
+  - pip:
+    - affine==2.2.1
+    - msgpack==0.5.6
+    - rasterio==1.0.8
+    - rio-mucho==1.0.0
+    - rio-toa==0.3.0
+    - snuggs==1.4.2
+prefix: /Users/jstevens/miniconda3/envs/nasa_confirm
+
```
Comment (jbednar) [inline environment.yml]: Shouldn't be fully pinned like this; just pin pyviz and what we are actually relying on directly; the underlying packages can pin their own versions if need be.

**Comment by jlstevens** [comment]
@jbednar I think this is now ready to merge.

**Pull request merged**